doc/datadog.md

```diff
@@ -0,0 +1,45 @@
+# Datadog credentials
+
+### What this is for
+
+When setting up an environment the Datadog credentials need to be decrypted and pushed into the S3 state bucket as they are required during pipeline runs. Each environment (dev, ci, staging, and prod) has its own set of Application and API credentials in the PaaS credential store.
+
+### Requirements
+
+* Make sure you have access to the PaaS credential store, this is required for Datadog credentials.
+* Load the AWS credentials for the environment you are setting up datadog for. These are required as the Datadog secrets file is stored in an S3 bucket.
+
+### Usage
+```
+make <ENV> upload-datadog-secrets
+```
+
+### Example for testing in a dev environment with standard dev API/application keys
+Upload the dev keys to your state bucket
+
+```
+make dev upload-datadog-secrets
+```
+
+### Example for testing in a dev environment with new API/application keys
```

Comment:
Is this section required? This looks like a very rare situation and I'm not sure it's worth complicating this doc.